pkg/nodebootstrap/legacy/maxpods.go

```diff
@@ -4,405 +4,499 @@ package legacy
 
 // Source: https://raw.github.com/awslabs/amazon-eks-ami/master/files/eni-max-pods.txt
 var maxPodsPerNodeType = map[string]int{
-	"a1.2xlarge":    58,
-	"a1.4xlarge":    234,
-	"a1.large":      29,
-	"a1.medium":     8,
-	"a1.metal":      234,
-	"a1.xlarge":     58,
-	"c1.medium":     12,
-	"c1.xlarge":     58,
-	"c3.2xlarge":    58,
-	"c3.4xlarge":    234,
-	"c3.8xlarge":    234,
-	"c3.large":      29,
-	"c3.xlarge":     58,
-	"c4.2xlarge":    58,
-	"c4.4xlarge":    234,
-	"c4.8xlarge":    234,
-	"c4.large":      29,
-	"c4.xlarge":     58,
-	"c5.12xlarge":   234,
-	"c5.18xlarge":   737,
-	"c5.24xlarge":   737,
-	"c5.2xlarge":    58,
-	"c5.4xlarge":    234,
-	"c5.9xlarge":    234,
-	"c5.large":      29,
-	"c5.metal":      737,
-	"c5.xlarge":     58,
-	"c5a.12xlarge":  234,
-	"c5a.16xlarge":  737,
-	"c5a.24xlarge":  737,
-	"c5a.2xlarge":   58,
-	"c5a.4xlarge":   234,
-	"c5a.8xlarge":   234,
-	"c5a.large":     29,
-	"c5a.metal":     737,
-	"c5a.xlarge":    58,
-	"c5ad.12xlarge": 234,
-	"c5ad.16xlarge": 737,
-	"c5ad.24xlarge": 737,
-	"c5ad.2xlarge":  58,
-	"c5ad.4xlarge":  234,
-	"c5ad.8xlarge":  234,
-	"c5ad.large":    29,
-	"c5ad.metal":    737,
-	"c5ad.xlarge":   58,
-	"c5d.12xlarge":  234,
-	"c5d.18xlarge":  737,
-	"c5d.24xlarge":  737,
-	"c5d.2xlarge":   58,
-	"c5d.4xlarge":   234,
-	"c5d.9xlarge":   234,
-	"c5d.large":     29,
-	"c5d.metal":     737,
-	"c5d.xlarge":    58,
```

Comment:
git doing a great job at rendering the diff here... whats actually changed?

Reply:
Cool new instance types of course!